src/core.js

```diff
@@ -178,7 +178,9 @@ QUnit.equiv = equiv;
 QUnit.dump = dump;
 
 // 3.0 TODO: Remove
-QUnit.jsDump = dump;
+Object.defineProperty( QUnit, "jsDump", {
+    get: function() { throw new Error( "Use QUnit.dump instead of QUnit.jsDump" ); }
```

Comment:
Let's update this to be consistent with [the other deprecations](https://github.com/jquery/qunit/blob/89731f969e6ab9201149c0a1d23767aec6ca72d9/src/export.js#L23-L24):

```
"QUnit.jsDump is removed in QUnit 2.0, use QUnit.dump instead.\n" +
"Details in our upgrade guide at https://qunitjs.com/upgrade-guide-2.x/"
```

The error should also be thrown on `set` (just in case).